source/conf.py

```diff
@@ -45,6 +48,7 @@
     "notfound.extension",
     "versionwarning.extension",
     "sphinx_panels",
+    "minify",
```

Comment:
It's not clear if this item is downloaded from pypi or is local (And I don't know how sphinx would resolve conflicts).
Could you change in-repo extensions to clearly denote that they are local and to prevent potential conflicts with pypi packages? Maybe create a separate `local_extensions` list that would only resolve in-repo extensions?